linux_os/guide/system/network/network-nftables/set_nftables_base_chain/ansible/shared.yml

```
@@ -0,0 +1,38 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+#Name of the table
+{{{ ansible_instantiate_variables("var_nftables_table") }}}
+#Familiy of the table 
+{{{ ansible_instantiate_variables("var_nftables_family") }}}
+#Name(s) of base chain
+{{{ ansible_instantiate_variables("var_nftables_base_chain_names") }}}
+#Type(s) of base chain
+{{{ ansible_instantiate_variables("var_nftables_base_chain_types") }}}
+# Hooks for base chain
+{{{ ansible_instantiate_variables("var_nftables_base_chain_hooks") }}}
+#Priority
+{{{ ansible_instantiate_variables("var_nftables_base_chain_priorities") }}}
+#Policy 
+{{{ ansible_instantiate_variables("var_nftables_base_chain_policies") }}}
+
+- name: {{{ rule_title }}} - Collect Existing NFtables
+  ansible.builtin.command: nft list table {{ var_nftables_family }} {{ var_nftables_table }}
+  register: existing_nftables
+  ignore_errors: true
+
+- name: {{{ rule_title }}} - Set NFTable Table
+  ansible.builtin.command: nft create table {{ var_nftables_family }} {{ var_nftables_table }}
+  when: existing_nftables.stdout_lines | length == 0
```

```suggestion
  when: existing_nftables.rc > 0
```
Actually, it could be simplified to only `when: existing_nftables.rc` but it would require the reader to previously know the `nft` return codes. Making it explicit improve the readability. I also think it is less prone to break in the future when testing the return code instead of stdout.

---

Hello @marcusburghardt 
With my last commit I have implemented the proposed change
All the best 
Rumen